config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,23 @@
+redis_url = "#{ENV["REDIS_URL"]}/0"
+
+options = {
+  concurrency: Integer(ENV.fetch("RAILS_MAX_THREADS") { 5 }),
+}
+
+Sidekiq.configure_server do |config|
+  config.options.merge!(options)
+  config.redis = {
+    url: redis_url,
+    size: config.options[:concurrency] + 5,
```

Comment:
Worth linking to https://github.com/mperham/sidekiq/wiki/Using-Redis#complete-control here - otherwise that `concurrency + 5` seem arbitrary?

Reply:
I'm happy to put it in the commit so that if someone wants to blame it it's available for reading.